src/msgpack.light/Converters/Generation/DuplicateArrayElementException.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace ProGaudi.MsgPack.Light.Converters.Generation
+{
+    public class DuplicateArrayElementException : Exception
```

Comment:
Let's create and use GeneratorException derived from some SerializationException